src/formatDate.js

```diff
@@ -8,7 +8,32 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const separator = fromFormat[3];
```

Comment:
The code assumes that the separator is always at index 3 of the 'fromFormat' array. This assumption might not hold true if the format changes. Consider making the separator dynamic or explicitly defined.